tests/test_association_task.py

```diff
@@ -75,13 +95,40 @@ def test_run(self):
                 [0]):
             self.assertEqual(test_obj_id, expected_obj_id)
 
+    def test_run_trailed_sources(self):
+        """Test the full task by associating a set of diaSources to
+        existing diaObjects when trailed sources are filtered
+        # """
+
+        config = AssociationTask.ConfigClass()
+        config.doLongTrailFilter = True
```

Comment:
Since it's the default, you don't need to set it here, or even create a config at all.